Generation/src/components/MomentumRangeParticleGun.h

```diff
@@ -12,8 +12,8 @@
  *
  *  Particle gun with given momentum range
  *
- *  @author Patrick Robbe
- *  @author Benedikt Hegner
+ *  @author Patrick Robbe (adaptation to tool structure)
```

Comment:
The original author adapted to tool structure? I guess that this was a mistake and you wanted to add that to your own name, which is missing from the list?